tests/test_schema.py

```diff
@@ -371,6 +371,17 @@ def test_schema_bad_prop_06(self):
             abstract link foo extending foo;
         """
 
+    @tb.must_fail(errors.InvalidReferenceError,
+                  "object type or alias 'std::str' does not exist"):
```

Comment:
Also, I think we should see if we can add a specific handler for this error to hint that `{` here is not the same as opening a DDL sub-block, i.e.

This is a shape:

```
property foo := <expr> {
   ...
}
```

should instead be

```
property foo {
  using (<expr>),
  ...
}
```

Reply:
This can be a separate PR of course.